lib/cli_host_controller.rb

```diff
@@ -37,8 +37,20 @@ def setup(url)
   def list
     host_config = HostConfig.for_directory(@config_dir)
     host_config.read
-    host_config.hosts.each do |host|
-      @out.puts host
+    host_config.hosts.each do |host_name|
+      host = host_config.host(host_name)
+      out = "#{host_name}"
+      attributes = []
+      if host['address']
+        attributes.push("address: #{host['address']}")
+      end
+      if host['base_snapshot_id']
+        attributes.push("base snapshot id: #{host['base_snapshot_id']}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.